nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -1634,7 +1635,7 @@ def _write_encoded_jaccard_pair(self, dedup_parsed_id_path):
             len(ddf_id)
             ddf = dask_cudf.read_parquet(
                 self.jaccard_pairs_path,
-                blocksize="256MB",
+                blocksize="1GB",
```

Comment:
We can probably scale this up a bit more (I kept it conservative). 

CC: @ayushdg

Reply:
Similar to above, can you write why we decrease the above blocksize but increased this?